checks.c

```diff
@@ -1798,8 +1798,10 @@ static void check_graph_child_address(struct check *c, struct dt_info *dti,
 		struct property *prop = get_property(child, "reg");
 
 		/* No error if we have any non-zero unit address */
-		if (prop && propval_cell(prop) != 0)
+		/* We have a check for val.len == sizeof(cell_t) elsewhere */
+                if (prop && propval_cell_n(prop, 0)  != 0 ) {
```

Comment:
Do you have a danger of a different assertion failure if the length of prop is < 4?

There's also an extraneous space here, before the `!=`.

Reply:
Since the case of prop length < 4 cannot simply be handled by using `propval_cell_n` (with the assertion fixed) instead of `propval_cell` I am reverting this change. Instead I'm adding `graph_port` and `graph_endpoint` as prerequisites for `graph_child_address`.

Another alternative would be to call `check_graph_reg()` in `check_graph_child_address()` directly but then any warnings in `check_graph_reg()` would be issued twice.